### DISCUSSION TITLE
To switch from camera to local video

### DISCUSSION THREAD

**Discussion by yuvaraj-ncompass** [HaishinKit/HaishinKit.swift#1008] [Q&A]
Hi, Is there a way I can stop livestreaming from the camera, and send a pre-recorded video instead?

**Comment by DrDepTraii** [comment]
Have you done it yet? I also need to deal with this problem

**Comment by shogo4405** [comment]
Close as duplicate https://github.com/shogo4405/HaishinKit.swift/discussions/1358
